ibm/data_source_ibm_is_instance_group_managers.go

```diff
@@ -101,7 +101,7 @@ func dataSourceIBMISInstanceGroupManagersRead(d *schema.ResourceData, meta inter
 
 	// Support for pagination
 	start := ""
-	allrecs := []vpcv1.InstanceGroupManager{}
+	allrecs := []vpcv1.InstanceGroupManagerIntf{}
```

Comment:
These all changes are already commited and merged please take the latest code and rebase your branch